src/contacts/ViewContact/ViewContactContainer.js

```diff
@@ -88,6 +96,11 @@ class ViewContactContainer extends Component {
 
     const contact = get(resources, 'contact.records[0]', {});
     const categories = get(resources, 'categories.records', []);
+    const addresses = hydrateAddresses(categories, contact.addresses);
+    const emails = mixCategories(categories, contact.emails);
+    const phoneNumbers = mixCategories(categories, contact.phoneNumbers);
```

Comment:
`mixCategories` is a piece of crap. If you're re-writing components please don't use it. IMHO, the best solution to have categories inside display components itself

Reply:
removed it too